types/google-apps-script/google-apps-script.base.d.ts

```diff
@@ -22,13 +22,13 @@ declare namespace GoogleAppsScript {
       copyBlob(): Blob;
       getAs(contentType: string): Blob;
       getBytes(): Byte[];
-      getContentType(): string;
+      getContentType(): string | null;
       getDataAsString(): string;
       getDataAsString(charset: string): string;
       getName(): string | null;
       isGoogleType(): boolean;
       setBytes(data: Byte[]): Blob;
-      setContentType(contentType: string): Blob;
+      setContentType(contentType: string | null): Blob;
```

Comment:
I tested Blob.setContentType accepts null argument in current Google Apps Script.
But [the api doc](https://developers.google.com/apps-script/reference/base/blob#setcontenttypecontenttype) which you write in description says the argument type is "String", not nullable.
I think it leaves it as it is, how do you think about it?

Reply:
> As they are getting/setting the same property?

No. getContentType may return null when **you pass null to contentType argument of Utilities.newBlob**.

The API doc says contentType of newBlob can be `string | null`.

> The content type of the blob - can be null.
https://developers.google.com/apps-script/reference/utilities/utilities#newBlob(String,String)

Reply:
I agree getContent returns `string | null`. It is same as api doc.

However the doc don't say the arg contentType of Blob.setContentType accepts `string | null`.
This is the different from the contentType of Utilities.newBlobs.

I know from [my experimentation](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66323#discussion_r1287818343) that it does not throw an error when pass null to setContentType. But that does not mean that the type definition is nullable, because the doc doesn't say that it is nullable.

Reply:
I don't know why you made it nullable which is not wrtten in the doc, so if you have any reason, please let me know.

Reply:
I work at google and our mocks assume that setContentType accepts a `string | null`. 
They do not match and that complicates testing. This change would help us solve that annoyance.

Reply:
All right, I approve this pull request.

I am sorry for my lack of understanding 🙇 
I maintenance this definition for free, and also I don't work at Google. All I can do is to check whether the changes of pull request are consistent with api doc.
It would be great if you could update the api doc of setContentType because it is not just googlers who use this type definition. Please give it some consideration 🙏

Reply:
Will do. I have made a request to update the documentation. Thanks!